application/CohortManager/src/Functions/Shared/Model/Participant.cs

```
@@ -109,6 +109,26 @@ public ParticipantDemographic ToParticipantDemographic()
         };
     }
 
+    public ParticipantManagement ToParticipantManagement()
+    {
+        var participantManagement = new ParticipantManagement
```

Can you throw an exception if we cannot parse the Participant Management Numbers to long?
I'd rather we errored here rather than insert bad data.

---

changed from TryParse to Parse